### DISCUSSION TITLE
2 channel Frontend with over/under-voltage protection

### DISCUSSION THREAD

**Discussion by charlee** [fhdm-dev/scoppy#32] [Scoppy Front end designs]
Hi all, I created a simulation of a 2 channel Frontend.
The design is based on [Design 3 example](https://oscilloscope.fhdm.xyz/wiki/front-end-design-3).
I only adjusted the resistor values based on what I have, and used the unused op-amp for the second channel.
This way, a single LM324N is sufficient.

In the design, channel 1 and channel 2 share the same `Vref` and the same op-amp for over-voltage protection.
My understanding is that, since the output of an ideal op-amp is an ideal voltage source,
it should be possible to connect multiple loads to the output of `OP1` without impacting each other.

In the simulation, I used two AC sources with diferent frequencies (2kHz vs. 1.37kHz)
and it turned out that `Vref` only has negligible changes.
And `ADC1` and `ADC2` do not affect each other.

Please let me know if I am completely wrong!

![image](https://user-images.githubusercontent.com/166152/138012479-3b4524c2-d055-46ff-9846-9cef2c2426d9.png)

**Comment by fhdm-dev** [comment]
Looks good to me! The disadvantage of using the same Vref op-amp for both channels is that the channels can't have different input voltage ranges - but if that's not an issue for you then it makes sense to use the same op-amp for both.
